src/game.js

```diff
@@ -24,24 +25,29 @@ var hasProp = {}.hasOwnProperty;
 var indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
 
 var blackBlank = "_____";
-var blackCards = require('./blackcards.coffee');
-var whiteCards = require('./whitecards.coffee');
 
 Game.prototype.init = function (data) {
   this.initialized = true;
+
   data.cah = data.cah || _.cloneDeep(defaultData);
   this.db = _.defaults(data.cah, defaultData);
 
   this.db.decks.ud = [];
+  this.db.modes = {
+    base: true
+  };
 
-  this.shuffle();
+  this.resetDecks();
 };
 
 Game.prototype.toggle_mode = function (mode) {
   this.db.modes = this.db.modes || {};
   this.db.modes[mode] = !this.db.modes[mode];
+  deck.setModes(this.db.modes);
+
+  this.resetDecks(); // adds/removes cards for toggled mode
```

Comment:
Yep. There is a chance that a previously played card could show up again soon. The alternative would be to finish the existing deck, then the new set would get pulled in. I like the immediate gratification 😄